storybook/storybook-docs/src/grid.stories.mdx

```diff
@@ -4,110 +4,153 @@ import { Meta } from "@storybook/blocks";
 
 # Grid
 
-Onder elke pagina ligt hetzelfde grid.
-Alle elementen op een pagina worden op het grid geplaatst.
-Ze beslaan dan een bepaald aantal kolommen, en de witruimtes daartussen.
+Onder elke pagina ligt een _responsive_ grid.
+Alle elementen op een pagina worden op dat grid geplaatst.
 
 ## Ontwerp
 
-Het grid is volledig _responsive_: afhankelijk van de breedte van het venster verandert de breedte van het grid en het aantal kolommen.
+### Drie kolombreedtes
```

Comment:
This is about the number of columns not the actual width of the column right? I would use something like "Drie grid varianten"